rask-engine/src/resources/resource_table.rs

```diff
@@ -83,14 +83,6 @@ impl Default for ResourceTable {
 }
 
 impl ResourceTable {
-    /// Create a new library initialized with None resources.
-    #[cfg(feature = "nightly")]
```

Comment:
Since rust 1.50 (see https://github.com/rust-lang/rust/releases/tag/1.50.0 the first change in the list), this is possible in stable rust.
So why did you delete the `const fn` approach?
Side fact:
With the feature `const_fn_transmute` activated, the `core::mem::transmute` approach works with `const fn`